components/ListHeader/list-header.tsx

```diff
@@ -78,7 +80,7 @@ const ListHeader = ({
         <Button onClick={() => handleCopyToClipboard()} className="px-6 py-2 bg-white " variant="text">
           <FiCopy className="mt-1 mr-2" /> Share
         </Button>
-        {isOwner && (
+        {isOwner && !overLimit && (
```

Comment:
We shouldn't hide the button if they are over the limit. Instead, show the modal when they click the button. 

![Screenshot 2024-06-04 at 5 23 58 PM](https://github.com/open-sauced/app/assets/5713670/97ef4303-893f-4b4f-8f7d-a6e593c5ab25)

Reply:
Fixed